src/main/java/roomescape/common/exception/AlreadyExistException.java

```diff
@@ -0,0 +1,10 @@
+package roomescape.common.exception;
+
+public class AlreadyExistException extends RuntimeException {
+    public AlreadyExistException() {
+    }
+
+    public AlreadyExistException(String message) {
+        super(message);
+    }
+}
```

Comment:
```suggestion
public class AlreadyExistException extends RuntimeException {
    public AlreadyExistException() {
    }

    public AlreadyExistException(String message) {
        super(message);
    }
    
    public static final class AlreadyExistMemberException extends AlreadyExistException {
        public AlreadyExistMemberException() {
        }
    
        public AlreadyExistMemberException(String message) {
            super(message);
        }
    }
}
```

inner 클래스로 생성한다면 관리하기 더욱 편해질 것 같아요! 분산된 클래스를 한곳에 모으는건 어때요?